zproject/backends.py

```diff
@@ -1488,8 +1628,6 @@ def get_verified_emails(self, *args: Any, **kwargs: Any) -> List[str]:
 @external_auth_method
 class SAMLAuthBackend(SocialAuthMixin, SAMLAuth):
     auth_backend_name = "SAML"
-    standard_relay_params = ["subdomain", "multiuse_object_key", "mobile_flow_otp", "desktop_flow_otp",
-                             "next", "is_signup"]
```

Comment:
Can you rebase?  This hunk will disappear from the diff as we've already merged this commit.